docs/components/modeler/web-modeler/play-your-process.md

```diff
@@ -194,3 +194,11 @@ See the table below for the requirement for each field, as well as an example va
 ### Limitations
 
 The environment variables `CAMUNDA_CUSTOM_CERT_CHAIN_PATH`, `CAMUNDA_CUSTOM_PRIVATE_KEY_PATH`, `CAMUNDA_CUSTOM_ROOT_CERT_PATH`, and `CAMUNDA_CUSTOM_ROOT_CERT_STRING` can be set in Docker or Helm chart setups. However, these configurations have not been tested with Play's behavior and, therefore, are not supported when used with Play.
+
+## Play Usage and Billing Considerations
+
+The use of Play may result in additional charges depending on your organization's plan and the type of cluster you are using. To avoid incurring extra costs, please follow these guidelines based on your plan:
+
+- **Trial/Internal Plans:** You can use any cluster.
```

Comment:
our users do not need to know that there is something like an internal plan.

also not sure if trial plan is actually what should be taken? isn't it rather called "Camunda SaaS Trial" by now? probably best for @felix-mueller to comment on

Reply:
removed the 'Internal Plan', waiting for @felix-mueller suggestion for the Trial 👀